lightning/src/routing/scoring.rs

```diff
@@ -1939,15 +1972,20 @@ ReadableArgs<(ProbabilisticScoringDecayParameters, G, L)> for ProbabilisticScore
 		r: &mut R, args: (ProbabilisticScoringDecayParameters, G, L)
 	) -> Result<Self, DecodeError> {
 		let (decay_params, network_graph, logger) = args;
-		let mut channel_liquidities = new_hash_map();
+		let mut channel_liquidities: HashMap<u64, ChannelLiquidity> = new_hash_map();
 		read_tlv_fields!(r, {
 			(0, channel_liquidities, required),
 		});
+		let mut last_duration_since_epoch = Duration::from_secs(0);
```

Comment:
The idea is that when there is no data, the first pathfinding operation works with time zero for all the penalty calculations?